installers/gladius-node-win-installer/install-script.iss

```diff
@@ -142,14 +143,17 @@ end;
 [Run]
 Filename: "{app}\gladius-networkd.exe"; Parameters:"install"; StatusMsg:"Installing GladiusNetworkDaemon as a service"; Flags: runascurrentuser runhidden
 Filename: "{app}\gladius-networkd.exe"; Parameters:"start"; Flags: runascurrentuser runhidden
-Filename: "{app}\gladius-controld.exe"; Parameters:"install"; StatusMsg:"Installing GladiusControlDaemon as a service"; Flags: runascurrentuser runhidden
-Filename: "{app}\gladius-controld.exe"; Parameters:"start"; Flags: runascurrentuser runhidden
+Filename: "{sys}\sc.exe"; Parameters: "create ""Gladius Control Daemon"" start=auto binPath= ""{app}\gladius-controld.exe"""; StatusMsg:"Installing Gladius Control Daemon as a service"; Flags: runhidden;
+Filename: "{sys}\sc.exe"; Parameters: "start ""Gladius Control Daemon""" ; Flags: runhidden
+
+Filename: "{sys}\sc.exe"; Parameters: "create ""Gladius Network Daemon"" start=auto binPath= ""{app}\gladius-networkd.exe"""; StatusMsg:"Installing Gladius Network Daemon as a service"; Flags: runhidden;
+Filename: "{sys}\sc.exe"; Parameters: "start ""Gladius Network Daemon""" ; Flags: runhidden

```

Comment:
This does not work to start the Network Daemon, nor does right clicking start in the service manager @alex-godwin